mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -92,7 +97,47 @@ def __init__(self, main_window: 'MainWindowView'):
         self.tableView.setSelectionMode(QAbstractItemView.SingleSelection)
         self.tableView.setAlternatingRowColors(True)
 
+        # Roi Prop table
+        self.roiPropertiesGroupBoxTitle: str = "Roi Properties: "
+        self.roi_table_properties = ["Top", "Bottom", "Left", "Right"]
+        self.roi_table_properties_secondary = ["Width", "Height", "Area"]
+        self.roi_table_properties_all = self.roi_table_properties + self.roi_table_properties_secondary
+        self.roiPropertiesTableWidget.setColumnCount(2)
+        self.roiPropertiesTableWidget.setRowCount(len(self.roi_table_properties_all))
+
+        self.roiPropertiesSpinBoxes = {}
+        self.roiPropertiesLabels = {}
+        row_ind = 0
+        for prop in self.roi_table_properties_all:
+            table_widget = QTableWidgetItem(prop)
+            table_widget.setFlags(Qt.ItemIsSelectable)
+            self.roiPropertiesTableWidget.setItem(row_ind, 0, table_widget)
+            if prop in self.roi_table_properties:
+                spin_box = QSpinBox()
```

Comment:
Can we decrease the width of the spin boxes? maybe by as much as half to free up the GUI a little bit?